doc/reduce-traffic.md

```diff
@@ -48,3 +48,7 @@ Be reminded of the effects of this setting.
   wallet is loaded or if you use the node to broadcast transactions.
 - It makes block propagation slower because compact block relay can only be
   used when transaction relay is enabled.
+- This setting sets the flag "-walletbroadcast" to be "0", if it is currently
+  set to "1", which disables the automatic broadcasting of transactions
```

Comment:
```suggestion
  unset, which disables the automatic broadcasting of transactions
```

The argument is set "softly", see also https://doxygen.bitcoincore.org/class_args_manager.html#ae77153300424d36e02e54f2c718a457f

Reply:
Ah got it. Thanks!

Reply:
I still have a clarifying question about "softset". My understanding is that this param interaction is not guranteed to happen. It would only happen if the setting was not manually set already by the user. 

Therefore, If the user sets -walletbroadcast to 1 first, and later sets -blocksonly to 0, this would effectively means that 1) -walletbroadcast will not be set to 0 and 2) all transaction broadcast happens normally?

Reply:
> Therefore, If the user sets -walletbroadcast to 1 first, and later sets -blocksonly to 0, this would effectively means that 1) -walletbroadcast will not be set to 0 and 2) all transaction broadcast happens normally?

Yes.